docs/_includes/components/tooltips/tooltips-interactive-demo.html

```diff
@@ -0,0 +1,69 @@
+<!-- =================================================
+BEGIN: Tooltips Interactive Demo
+================================================== -->
+
+<section id="tooltips-interactive-demo">
+
+  <h2>
+
+    Interactive Demo
+
+  </h2>
+
+  <p>
+
+    You can include more than body text in a message. Messages can support any HTML content. All type styling will be retained. However, if you add the class <code>.message-type-styling</code> to any typographic element, the preferred message type color and styling will be applied. For links, add the class <code>.message-link</code> to any <code>&lt;a&gt;</code>.
+
+  </p>
+
+  <!-- =================================================
+  BEGIN: Example
+  ================================================== -->
+
+  <div class="panel flush-bottom">
+
+    <div class="panel-cell">
+
+      <button type="button" class="button" data-toggle="tooltip" data-placement="top" title="Tooltip on top">
+
+        Tooltip on top
+
+      </button>
+
+      <button type="button" class="button" data-toggle="tooltip" data-placement="right" title="Tooltip on right">
+
+        Tooltip on right
+
+      </button>
+
+      <button type="button" class="button" data-toggle="tooltip" data-placement="bottom" title="Tooltip on bottom">
+
+        Tooltip on bottom
+
+      </button>
+
+      <button type="button" class="button" data-toggle="tooltip" data-placement="left" title="Tooltip on left">
+
+        Tooltip on left
+
+      </button>
+
+    </div>
+
+    <div class="panel-cell panel-cell-light panel-cell-code-block">
+
+<pre class="prettyprint transparent flush lang-html">
+&lt;div class="tooltip tooltip-top"&gt;
```

Comment:
Not sure if this is showing the correct example. Expected it to show me markup for the examples above (i.e. <button...>)